rdrf/rdrf/admin.py

```diff
@@ -452,6 +453,23 @@ class DashboardWidgetAdmin(admin.ModelAdmin):
     inlines = [DashboardLinksInline, DashboardDemographicsInline, DashboardCdeDataInline]
 
 
+class LongitudinalFollowupAdmin(admin.ModelAdmin):
+    model = LongitudinalFollowup
+
+    list_display = (
```

Comment:
Is it worth adding some validation to ensure that an email notification exists for the registry that relates to the context form group selected when creating/editing a `LongitudinalFollowup`?

Reply:
Closing as per discussion in dev meeting.